src/patternfly/components/Menu/menu.scss

```diff
@@ -282,7 +282,11 @@
       // target first list in menu
       > .#{$menu}__content > .#{$menu}__list,
       > .#{$menu}__list {
-        transform: translateX(-100%);
+        @include pf-v5-bidirectional-style(
+          $prop: transform,
+          $ltr-val: translateX(-100%),
+          $rtl-val: translateX(100%)
```

Comment:
For these places you're using `pf-v5-bidirectional-style` where you want the inverse of some value in RTL, if you use the [`pf-v5-calc-invserse` function](https://github.com/patternfly/patternfly/blob/447630c3449372332ee1628268397f16f5ffa965/src/patternfly/sass-utilities/functions.scss#L121-L123), that will allow the translate to go back to normal if this component is used in a block set to [`.pf-v5-m-dir-ltr`](https://github.com/patternfly/patternfly/blob/447630c3449372332ee1628268397f16f5ffa965/src/patternfly/base/_common.scss#L36-L42) since the inverse value is tied to `--#{$pf-global}--inverse--multiplier`

The example we have of this is with the switches in the bottom right of the full page examples. The little circles in the switch use `translateX()`, and use `pf-v5-calc-invserse` to control their position in RTL. The doc is RTL, but that box is set to `.pf-v5-m-dir-ltr`, which resets `--#{$pf-global}--inverse--multiplier`, so the `translateX()` is reset.